spyder/app/tests/test_mainwindow.py

```diff
@@ -5322,14 +5323,209 @@ def test_frames_explorer(main_window, qtbot):
 
     assert len(frames_browser.frames) == 1
     assert list(frames_browser.frames.keys())[0] == "pdb"
-    assert not postmortem_debug_action.isEnabled()
+    assert not enter_debug_action.isEnabled()
 
     # Crash Kernel
     with qtbot.waitSignal(shell.sig_prompt_ready, timeout=10000):
         shell.execute("import ctypes; ctypes.string_at(0)")
 
     assert frames_browser.frames is None
-    assert not postmortem_debug_action.isEnabled()
+    assert not enter_debug_action.isEnabled()
+
+
+@pytest.mark.slow
+@flaky(max_runs=3)
+def test_executing_frames_explorer(main_window, qtbot):
+    """Test frames explorer"""
```

Comment:
```suggestion
def test_enter_debugger(main_window, qtbot):
    """
    Test that we can enter the debugger while code is running in the kernel.
    """
```

I'm not entirely sure about this suggestion but it seems to describe better what you're doing here.